### PR TITLE
Detail vote error.

### LINKED ISSUE #5: Improve messages regarding inability to vote
Instead of a generic, can't vote. It would be nice to explain why:
- Election hasn't started yet
- Election vote time is over
- Vote already registered

### PR DESCRIPTION
**Pull request by Kineolyan**
This PR provides an improved version of the flash error message generate when a vote is submitted before the election start or after the election end.

Again, the messages are not translated in Brazilian.

Closes: #5

### CODE REVIEW

File: resources/pt_BR.clj [hunk 1]
```diff
@@ -40,13 +40,14 @@
...
     :place "Votar"
     :used-token "Seu token já foi utilizado"
-    :not-accepting-votes "Essa eleição não está em período de votação"
     :recorded "Voto contabilizado! Obrigado."
     :invalid "Voto inválido. Por favor, verifique que selecionou a quantidade correta de candidatos."
     :confirm "Votos não podem ser alterados uma vez que forem enviados. Você tem certeza que deseja enviar seu voto?"
     :voter-count "Total de eleitores"
     :votes-count "Total de votos a receber"
     :casted-votes-count "Total de votos recebidos"
+    :not-started "Election has not started yet."
```
Comment (hugocorbucci) [inline resources/pt_BR.clj]: A eleição ainda não começou

File: resources/pt_BR.clj [hunk 2]
```diff
@@ -40,13 +40,14 @@
...
+    :too-late "Election vote time is over."
```
Comment (hugocorbucci) [inline resources/pt_BR.clj]: A eleição já encerrou

File: test/clj/election/controllers/votes_test.clj [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  (:require
+    [election.controllers.votes :as v]
+    [clj-time.core :as t]
+    [clj-time.coerce :as c]
+    [election.i18n.messages :as i18n]
+  )
+)
+
+
+; (defn- check-election-start [election]
```
Comment (hugocorbucci) [inline test/clj/election/controllers/votes_test.clj]: You can probably get rid of those now

**Review by hugocorbucci — CHANGES_REQUESTED** [review]
The translations and some cleanup on commented out code. Otherwise looks great. Thank you again @Kineolyan

File: test/clj/election/controllers/votes_test.clj (hunk 3 above)
Comment (Kineolyan) [reply test/clj/election/controllers/votes_test.clj]: I actually wanted to be thorough but it is true that this is very low level unit tests.

Comment (Kineolyan) [reply test/clj/election/controllers/votes_test.clj]: Oops my bad. I though you were talking about the unit tests and not the comments.

**Pull request merged**